pkgs/development/libraries/libdeltachat/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, perl
+, pkg-config
+, rustPlatform
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libdeltachat";
+  version = "1.52.0";
+
+  src = fetchFromGitHub {
+    owner = "deltachat";
+    repo = "deltachat-core-rust";
+    rev = version;
+    sha256 = "0im2wmnxjw6j09aca68ma1fn23vr728gp41c9clp3jf08bcfnfzm";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "0qfja0qkq976vk3210qbrm3820d1v3zvzra2387dyphyswqjqjm6";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    perl
+    pkg-config
+  ] ++ (with rustPlatform; [
+    cargoSetupHook
+    rust.cargo
+    rust.rustc
+  ]);
+
+  buildInputs = [
+    openssl.dev
```

Comment:
Isn't this done automatically?

Reply:
True, `.dev` is superfulous.

Reply:
Actually it's not needed at all. I don't know why I put it there.

Reply:
@SuperSandro2000 Do you know where this is documented?

Reply:
IIRC, as part of the multiple outputs work, it essentially do a `lib.getDev` on anything in buildInputs.